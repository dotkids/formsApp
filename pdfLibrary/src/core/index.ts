export * from 'core/pdf-document';
export * from 'core/pdf-objects';
export { default as PDFOperators } from 'core/pdf-operators';
export {
  default as PDFTextObject,
} from 'core/pdf-operators/text/PDFTextObject';
export * from 'core/pdf-parser';
export * from 'core/pdf-structures';
