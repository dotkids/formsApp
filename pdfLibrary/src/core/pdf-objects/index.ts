export { default as PDFArray } from './PDFArray';
export { default as PDFBoolean } from './PDFBoolean';
export { default as PDFDictionary } from './PDFDictionary';
export { default as PDFHexString } from './PDFHexString';
export { default as PDFIndirectObject } from './PDFIndirectObject';
export { default as PDFIndirectReference } from './PDFIndirectReference';
export { default as PDFName } from './PDFName';
export { default as PDFNull } from './PDFNull';
export { default as PDFNumber } from './PDFNumber';
export { default as PDFObject } from './PDFObject';
export { default as PDFStream } from './PDFStream';
export { default as PDFRawStream } from './PDFRawStream';
export { default as PDFString } from './PDFString';
