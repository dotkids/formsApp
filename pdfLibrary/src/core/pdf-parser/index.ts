export { default as parseDocument } from './parseDocument';
export { default as PDFParser } from './PDFParser';
