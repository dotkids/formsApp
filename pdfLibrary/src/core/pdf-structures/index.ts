import * as PDFXRef from './PDFXRef';

export { default as PDFCatalog } from './PDFCatalog';
export { default as PDFContentStream } from './PDFContentStream';
export { default as PDFObjectStream } from './PDFObjectStream';
export { default as PDFHeader } from './PDFHeader';
export { default as PDFPage } from './PDFPage';
export { default as PDFPageTree } from './PDFPageTree';
export { default as PDFTrailer } from './PDFTrailer';
export { default as PDFLinearizationParams } from './PDFLinearizationParams';
export { default as PDFXRefStream } from './PDFXRefStream';
export { PDFXRef };
