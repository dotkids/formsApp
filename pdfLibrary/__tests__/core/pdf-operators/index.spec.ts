// This is required to prevent an error due to circular imports in this test
import 'core/pdf-objects';

import PDFOperators from 'core/pdf-operators';

describe(`index`, () => {
  it(`matches the snapshot`, () => {
    expect(PDFOperators).toMatchSnapshot();
  });
});
