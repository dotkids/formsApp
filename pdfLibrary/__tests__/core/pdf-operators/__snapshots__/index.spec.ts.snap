// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index matches the snapshot 1`] = `
Object {
  "B": [Function],
  "CS": [Function],
  "Do": [Function],
  "F": [Function],
  "G": [Function],
  "J": [Function],
  "K": [Function],
  "M": [Function],
  "Q": [Function],
  "RG": [Function],
  "S": [Function],
  "SC": [Function],
  "SCN": [Function],
  "T": Object {
    "asterisk": [Function],
  },
  "TD": [Function],
  "TJ": [Function],
  "TL": [Function],
  "Tc": [Function],
  "Td": [Function],
  "Tf": [Function],
  "Tj": [Function],
  "Tm": [Function],
  "Tr": [Function],
  "Ts": [Function],
  "Tw": [Function],
  "Tz": [Function],
  "W": [Function],
  "b": [Function],
  "c": [Function],
  "cm": [Function],
  "cs": [Function],
  "d": [Function],
  "f": [Function],
  "g": [Function],
  "gs": [Function],
  "h": [Function],
  "i": [Function],
  "j": [Function],
  "k": [Function],
  "l": [Function],
  "m": [Function],
  "n": [Function],
  "q": [Function],
  "quote": Object {
    "double": [Function],
    "single": [Function],
  },
  "re": [Function],
  "rg": [Function],
  "ri": [Function],
  "s": [Function],
  "sc": [Function],
  "scn": [Function],
  "v": [Function],
  "w": [Function],
  "y": [Function],
}
`;
